vignettes/get_started.Rmd

```diff
@@ -0,0 +1,101 @@
+---
+title: "Get Started"
+output: 
+  rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Get Started}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Purpose of admiralci
+
+This repository contains GitHub Actions continuous integration/continuous delivery (CI/CD) workflows, most of which are used by [`Admiral`](https://github.com/pharmaverse/admiral) and its extensions.
+Workflows defined here are responsible for assuring high package quality standards without compromising performance, security, or reproducibility.
+
+Please refer to the [`.github/workflows`](.github/workflows) directory to view the source code for the GitHub Actions workflows.
+
+Notes :
+
+- Some workflows are using github actions from [InsightEngineering](https://github.com/insightsengineering/) group.
```

Comment:
typo: "InsightEngineering" instead of "InsightsEngineering"

Reply:
```suggestion
- Some workflows are using github actions from [InsightsEngineering](https://github.com/insightsengineering/) group.
```